doc/containers.md

```diff
@@ -172,7 +172,7 @@ host\_name              | string    | randomly assigned | no        | bridged, p
 hwaddr                  | string    | randomly assigned | no        | all                           | -             | The MAC address of the new interface
 mtu                     | integer   | parent MTU        | no        | all                           | -             | The MTU of the new interface
 parent                  | string    | -                 | yes       | physical, bridged, macvlan    | -             | The name of the host device or bridge
-vlan                    | integer   | -                 | no        | macvlan                       | network\_vlan | The VLAN ID to attach to
+vlan                    | integer   | -                 | no        | macvlan, physical             | network\_vlan | The VLAN ID to attach to
```

Comment:
Well, only kinda. The added weirdness with VLAN on top of physical is that you can have up to 4096 containers sharing that physical device, so long as they each attach to a different VLAN.
The main device (untagged) will also still be on the host.

There's also the problem that if you have two containers, one attached to vlan 100 and the other wanting the untagged vlan, you now need to start them in the right order. First the vlan 100 container, then the untagged one, otherwise the main physical device will be gone from the host and you can't create the vlan 100 device.

That's different from the regular physical case where attaching a container to the whole physical device then removes it from the host entirely.